src/client/visualizations/line-chart/line-chart-legend/line-chart-legend.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
```

Comment:
:)

Reply:
Once again, it's a rename. But all of the code is new, so we probably we should delete this line.